nixos/modules/services/databases/mysql.nix

```diff
@@ -263,7 +263,7 @@ in
             if ! test -e ${cfg.dataDir}/mysql; then
                 mkdir -m 0700 -p ${cfg.dataDir}
                 chown -R ${cfg.user} ${cfg.dataDir}
-                ${mysql}/bin/mysql_install_db ${installOptions}
+                ${mysql}/bin/mysql_install_db --defaults-file=/etc/my.cnf ${installOptions}
                 touch /tmp/mysql_init
```

Comment:
I wonder why it has this touch command in `/tmp`. It is not used anywhere.

Reply:
/tmp/mysql_init used in post-start mysql script. Need to preinstall sql databases.